Source/Krypton Components/Krypton.Toolkit/Converters/ImageLookupConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿#region BSD License
+/*
+ * 
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
```

Comment:
If this is a new file, then the License header needs to change

Reply:
Will be deleting this file, found another way.

Reply:
On second thought, keeping this file, but changing licence header.